definitions/procedures/pulp/remove.rb

```diff
@@ -0,0 +1,99 @@
+module Procedures::Pulp
+  class Remove < ForemanMaintain::Procedure
+    metadata do
+      description 'Remove pulp2'
+      for_feature :pulp2
+
+      confine do
+        check_min_version('katello-common', '4.0')
+      end
+    end
+
+    def sys
+      ForemanMaintain::Utils::SystemHelpers.new
+    end
+
+    def pulp_data_dirs
+      pulp_data_dir = feature(:pulp2).data_dir
+      [
+        pulp_data_dir + '/published',
+        pulp_data_dir + '/content',
+        pulp_data_dir + '/importers',
+        pulp_data_dir + '/uploads',
+        feature(:mongodb).data_dir
```

Comment:
I'm getting 'undefined method `data_dir' for nil:NilClass' on this line, i think its supposed to be 'mongo'.

However, i realized that both this feature adn the pulp2 feature will return nil if the 'pulp-server' rpm is not installed, as the features will not be 'enabled'.   Maybe we should just hard code those two directories here?  (they are hard coded in their respective features)